vampire_survivors/src/character.py

```
@@ -0,0 +1,8 @@
+import pygame
```

Ruff (0.12.2)

`pygame` imported but unused

Remove unused import: `pygame`

(F401)